lsi/rom/tools/cp16mic.py

```diff
@@ -0,0 +1,981 @@
+#!/usr/bin/python3
+#
+# Assembler for Western Digital CP16xx Microcode
+# Copyright (c) 2020 Viacheslav Ovsiienko <1801BM1@gmail.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Mnemonics table is borrowed from:
+#   Disassembler for Western Digital CP16xx/WD21xx Microcode
+#   Copyright 2015 Eric Smith <spacewar@gmail.com>
+#
+import os
+import re
+import sys
+import argparse
+
+MIN_LC = 0x0000
+MAX_LC = 0x0800
+
+T_SYM = 0
+T_REG  = 1
```

Comment:
`pep8` will issue a warning on extra space before `=` there. In fact `pep8` generates a lot of warnings related to spaces in the code

Reply:
OK, will run PEP-8 either